homeassistant/components/date/__init__.py

```diff
@@ -80,13 +80,13 @@ class DateEntity(Entity, cached_properties=CACHED_PROPERTIES_WITH_ATTR_):
     @final
     def device_class(self) -> None:
         """Return the device class for the entity."""
-        return None
+        return None  # noqa: RET501
```

Comment:
https://github.com/astral-sh/ruff/pull/12243 was accepted and merged.
Can you please remove all the changes to properties from this PR?